### ISSUE TITLE
List of attibutes value 

### ISSUE THREAD

**Issue by orovellotti** [PnEcrins/FollowDem-admin#4]
How do we manage the list of attributes values.

Do we coma seperate them and generate the drop down list from the coma seperate list ?

**Comment by camillemonchicourt** [comment]
Yes, I think it was the idea as defined here : https://github.com/PnEcrins/FollowDem/issues/48#issuecomment-405620938

> La table lib_attributes permet de définir des attributs additionnels pour chaque animaux. Leur type permet de définir leur affichage dans l'admin (select, radio, text...). Le champs value_list permet de prédéfinir une liste pour les attributs de type select. Le champs order permet de définir leur ordre d'affichage dans l'admin.
